Releasing.md

```diff
@@ -0,0 +1,30 @@
+Release process.
+
+Only for Maintainers.
+
+1. Tag with version to be released eg: git tag -a 0.4.0-beta -m ".4.0.0-beta"
+   git push origin 0.4.0-beta
+
+2. Submit a PR and merge it. (any harmless PR) To trigger CI for master. (no
```

Comment:
hm... can we just configure CI to build from tags?

Reply:
this section will be removed. We already did this part using the CI :)

Reply:
just removed this section.